mvp-spec/dsp-api.md

```diff
@@ -82,14 +89,15 @@ rotations.
 
 ## Definitions
 
-A **Transaction** is the sending of SWAN Data from the Root Party through the SWAN ecosystem by consecutive Transmissions.
+A **Transaction** is the sending of SWAN Data from the Root Party through the 
```

Comment:
"SWAN data"?

Reply:
I fix it in this PR: https://github.com/criteo/addressable-network-proposals/pull/51/files